docs/DeveloperGuide.adoc

```diff
@@ -782,51 +782,118 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new task |
+
+|`* * *` |user |add a priority level to a task |allow the application to order tasks by priority level
+
+|`* * *` |user |add a deadline to a task |allow the application automate management of my task priorities
+
+|`* *` |user |add a subtask to a task |split tasks into smaller subtasks for easier management
+
+|`* *` |user |add tags to a task |organize the tasks based on groups such as modules or projects
+
+|`* * *` |user |add a description to a task |give a task a description that cannot be sufficiently represented by name or subtasks
+
+|`* * *` |user |delete a task |remove a task
+
+|`* * *` |user |delete a deadline from a task |remove a task's deadline when necessary
+
+|`* *` |user |delete a subtask from a task |remove a subtask I no longer need
+
+|`* *` |user |delete a tag from a task |remove a tag from a task that no longer belongs to a group
+
+|`* *` |user |delete a description from a task|remove a description that is no longer relevant
+
+|`* *` |user |delete all tasks |remove all tasks quickly
+
+|`* * *` |user |edit a task's name |update the task's name in case of changes needed
+
+|`* * *` |user |edit a task's priority level |change a task's priority level when I need to reorganize priorities
+
+|`* * *` |user |edit a task's deadline |change a task's deadline
+
+|`* *` |user |edit a subtask of a task|change a subtask's name
+
+|`* *` |user |edit a task's description|change a task's description
+
+|`* * *` |user |mark a task as complete |indicate to the application to push a task to the completed list
+
+|`* *` |user |mark a subtask as complete |keep track of subtasks already done
+
+|`* * *` |user |view all uncompleted tasks |see all tasks that need to be done, along with the deadlines and priorities
+
+|`* * *` |user |view all details of a task |view all details of a specified tasks, after seeing a list of all tasks
+
+|`* * *` |user |view all completed tasks |see all tasks that have been completed, for past reference
+
+|`* *` |user with many tasks in the to-do list |view all tasks with a specific tag |locate all tasks with a specific tag, without the need of a manual search
+
+|`* *` |user with many tasks in the to-do list |view all tasks with a certain deadline |locate all tasks with a certain deadline, without the need of a manual search
+
+|`* *`|user |view a calender showing task's deadlines|see a chronic overview of all tasks and their deadlines
+
+|`* *` |user with many tasks in the to-do list |find a task by name|locate a task without the need to search through a list
+
+|`* *` |user with many tasks in the to-do list |find a task by description|locate a task when I have forgotten it's name
+
+|`* *` |user |have all tasks' priority levels be automatically updated as the deadline approaches |transfer the responsibility of updating priorities to the application
```

Comment:
Same as the comment provided above, it seems that the `So that I can` portion does not flesh out the solution to the user's problem.
Instead the user story at line 863 seems to also solve the user's problem for line 861, perhaps both user stories can be collapsed into 1 user story?